rts/lra-examples/cdi-participant/README.md

```diff
@@ -5,44 +5,26 @@ of an API that enables loosely coupled services to coordinate long running
 activities in such a way as to guarantee a globally consistent outcome without the
 need to take long duration locks on data.
 
-The cdi example takes a normal WAR and wraps it into a -thorntail runnable jar
-using the `thorntail-maven-plugin`:
-
-```
-    <plugin>
-      <groupId>io.thorntail</groupId>
-      <artifactId>thorntail-maven-plugin</artifactId>
-      <executions>
-        <execution>
-          <goals>
-            <goal>package</goal>
-          </goals>
-        </execution>
-      </executions>
-    </plugin>
-```
+The cdi example is a simple JAX-RS application with one of its methods annotated with an `@LRA` annotation and other methods annotated with `@Compensate` and `@Complete`. The `@Compensate` method will be invoked by the coordinator when the LRA is closed.
```

Comment:
The sentence sounds me wrong. The `@Compensate` will be invoked when LRA is canceled.

Reply:
+1